src/overlays/actors/ovl_En_Niw_Lady/z_en_niw_lady.c

```diff
@@ -20,43 +36,535 @@ const ActorInit En_Niw_Lady_InitVars = {
+            case 25:
+            case 26:
+            case 27:
+            case 28:
+                frames = SkelAnime_GetFrameCount(&D_06000718);
+                SkelAnime_ChangeAnim(&this->skelAnime, &D_06000718, 1.0f, 0.0f, frames, 0, -10.0f);
+                break;
+            case 100:
+                frames = SkelAnime_GetFrameCount(&D_0600A630);
+                SkelAnime_ChangeAnim(&this->skelAnime, &D_0600A630, 1.0f, 0.0f, frames, 0, -10.0f);
+                this->unk_276 = 0;
+                break;
+        }
+        return;
+    }
+}
+
+void func_80AB9F24(EnNiwLady* this, GlobalContext* globalCtx) {
+    s32 pad;
+
+    if ((Object_IsLoaded(&globalCtx->objectCtx, this->ObjectAneIndex) != 0) &&
+        (Object_IsLoaded(&globalCtx->objectCtx, this->ObjectOsAnimeIndex) != 0)) {
+        gSegments[6] = PHYSICAL_TO_VIRTUAL(globalCtx->objectCtx.status[this->ObjectAneIndex].segment);
+        SkelAnime_InitSV(globalCtx, &this->skelAnime, &D_060000F0, NULL, &this->limbDrawTable,
+                         &this->transitionDrawTable, 16);
+        gSegments[6] = PHYSICAL_TO_VIRTUAL(globalCtx->objectCtx.status[this->ObjectOsAnimeIndex].segment);
+        this->unk_27E = 1;
+        this->actor.gravity = -3.0f;
+        Actor_SetScale(&this->actor, 0.01f);
+        ActorShape_Init(&this->actor.shape, 0.0f, &ActorShadow_DrawFunc_Circle, 20.0f);
+        Collider_InitCylinder(globalCtx, &this->collider);
+        Collider_SetCylinder(globalCtx, &this->collider, &this->actor, &sCylinderInit);
+        this->unk_272 = 0;
+        this->actor.unk_1F = 6;
+        this->actor.draw = EnNiwLady_Draw;
+        switch (this->unk_278) {
+            case 0:
+                if (!(gSaveContext.itemGetInf[0] & 0x1000) && LINK_IS_CHILD) {
+                    SkelAnime_ChangeAnim(&this->skelAnime, &D_0600A630, 1.0f, 0.0f,
+                                         (s16)(f32)SkelAnime_GetFrameCount(&D_0600A630), 0, 0.0f);
+                } else {
+                    SkelAnime_ChangeAnim(&this->skelAnime, &D_060007D0, 1.0f, 0.0f,
+                                         (s16)(f32)SkelAnime_GetFrameCount(&D_060007D0), 0, 0.0f);
+                }
+                if (LINK_IS_ADULT) {
+                    this->actionFunc = func_80ABA778;
+                } else {
+                    this->actionFunc = func_80ABA21C;
+                }
+                return;
+            case 1:
+                SkelAnime_ChangeAnim(&this->skelAnime, &D_060007D0, 1.0f, 0.0f,
+                                     (s16)(f32)SkelAnime_GetFrameCount(&D_060007D0), 0, 0.0f);
+                this->actionFunc = func_80ABAD38;
+                return;
+        }
+    }
+}
+
+void func_80ABA21C(EnNiwLady* this, GlobalContext* globalCtx) {
+    this->actor.textId = EnNiwLadyMissingNiwTextIDs[0];
+    this->unk_262 = 6;
+    this->actionFunc = func_80ABA244;
+}
+
+void func_80ABA244(EnNiwLady* this, GlobalContext* globalCtx) {
+    EnNiw* firstNiw;
+    EnNiw* currentNiw;
```

Comment:
doesnt matter for now but i would personally just call them cuccos in temps and stuff